stock_prices/stock_prices.py

```diff
@@ -3,13 +3,58 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+    index = len(prices)
+    highest = -999999999999
```

Comment:
jaja very nice Chad! I think the line below will fit your needs
n_inf = float("-inf")

(2 min read)
https://medium.com/@sanatinia/how-to-work-with-infinity-in-python-337fb3987f06